script/Ciel/Day 6/06_01_CIEL06_6.txt

```diff
@@ -345,12 +345,12 @@ My immortality would cease at the moment of their destruction.'
 [sha:8ce989e371c8f8fd493e758d4da9e83017a7d2a0]{
 -- Page 247, Offset 13805.
 -- 『……ハ。
-'Hmm...
+'Hmph...
 }
 [sha:2ba89800dc20b3a84fbdf248bcf5361580475033]{
 -- Page 247, Offset 13806.
 -- 始まりからして終わりを見定めた不老不死とはな。
-An immortality whose end is known from the very beginning.
+Immortality with an end known from the very beginning?
```

Comment:
Hrm something about this one bothers me but will have to come back to it

Reply:
I think it's better than the original at least